src/pages/UsersTasks.jsx

```diff
@@ -49,8 +48,8 @@ export default class UsersTasks extends React.Component {
 
   async updateData() {
     const { userId } = this.state;
-    const user = await getElementFromCollection(USERS, userId);
-    const tasksWithStatus = await user.data().tasks;
+    const userData = await getElementDataFromCollection(USERS, userId);
+    const tasksWithStatus = await userData.tasks;
```

Comment:
why do you need to `await` here ? it looks like wrong code

Reply:
oOps, you are right! I really don't need it here:
```suggestion
    const userData = await getElementDataFromCollection(USERS, userId);
    const tasksWithStatus = userData.tasks;
```